### PR TITLE
Split drinks into different files

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Why It's Good For The Game Pull Request
Splits our nearly 4000 and 1600 line long files. Mostly just moves the glass appearances into their own files. 

## Changelog
:cl: Tattle
image: removed a stray gray pixel from the milk containers
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drinks/glass_styles/alcohol.dm [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+	desc = "A fine drink originally made to prevent waste by using the leftovers from winemaking."
+	icon_state = "grappa"
+
+/datum/glass_style/drinking_glass/amaretto
+	required_drink_type = /datum/reagent/consumable/ethanol/amaretto
+	name = "glass of amaretto"
+	desc = "A sweet and syrupy looking drink."
+	icon_state = "amarettoglass"
+
+
```
Comment (dragomagol) [inline code/modules/reagents/chemistry/reagents/drinks/glass_styles/alcohol.dm:99]:
```suggestion

```

**Review by Wallemations — APPROVED** [review]
That's a good job removing that pixel.

**Comment by carshalash** [comment]
DRAG. WHY DIDNT YOU PING ME.

**Review by Twaticus — CHANGES_REQUESTED** [review]
that pixel was soulful

**Pull request merged**